src/applications/gi/actions/index.js

```diff
@@ -184,11 +184,34 @@ export function fetchProfile(facilityCode, version) {
     dispatch({ type: FETCH_PROFILE_STARTED });
 
     return fetch(url, api.settings)
-      .then(res => res.json())
-      .then(
-        payload => withPreview(dispatch, { type: FETCH_PROFILE_SUCCEEDED, payload }),
-        err => dispatch({ type: FETCH_PROFILE_FAILED, err })
-      );
+      .then(res => {
+        if (res.ok) {
+          return res.json();
+        }
+
+        return res.json()
+          .then((errors) => {
+            throw new Error(errors);
+          });
+      })
+      .then(payload => {
+        const institutionZIP = _.get(payload, 'data.attributes.zip');
+        const bahUrl = `${api.url}/zipcode_rates/${institutionZIP}`;
+
+        return fetch(bahUrl, api.settings)
+          .then(res => res.json())
+          // if there's an error from the zipRatesPayload the reducer will just use the values from the institution end point.
```

Comment:
Decided not to error here since there's no resolution for a missing zip rate for a zip provided by the institution end point.